common/build/eslint-config-fluid/base.js

```diff
@@ -153,7 +153,13 @@ module.exports = {
 			},
 		],
 		"import/no-unused-modules": "error",
-		"import/order": "off",
+		"import/order": [
+			"error",
+			{
+				"newlines-between": "always",
+				"alphabetize": { order: "asc", caseInsensitive: true },
```

Comment:
Need to make a call about `caseInsensitive`. Stylistically I prefer `true`, however the default is `false` and the current Biome organization used sensitive sort. If we are going to go to case insensitive, then I think it should be a secondary change to make the adoption churn be minimal and only about blank lines (or places that missed Biome reorg).

Reply:
Adopting changes incrementally is exceedingly time consuming and costly with our current lint setup. The config is published and consumed that way so it is not easy to make a change then adjust it quickly. We have been trying to use local references to the lint config but given how eslint looks up plugins and how pnpm install deps for linked dependencies (it doesn't), it hasn't been trivial to do that. One reason we have multiple configs is to help incremental adoption without having to publish a new version of the config for each change.

Reply:
So if we need to adopt this incrementally, we should create configs that reflect the increments we want to take so we don't have to republish and re-ingest every time.

Reply:
When I was thinking about making this adjustment, I was considering placing the `newlines-between` setting in one of the stricter settings.
*I* would be completely fine if `caseInsensitive` is never made `true`. Too bad Biome doesn't have that option since it is fast and would save the fix time in regard to that.
If we are making the best and thoughtful changes when we adjust things, then I don't think we worry about the time to execute because we won't change too often. I'm good with whatever decision here so long as we aren't changing our minds and incurring that cost.

Reply:
I think we applied to client group with `caseInsensitive: false` in pre-pass. Do we still want `true` here? I am going to approve for either case. (no pun intended)

Reply:
I'm on the fence. I see the pros/cons this way:

`caseInsensitive: false` PROS:

1. Already mostly sorted that way, so less churn.
2. Compatible with Biome's sorting (but this is not a very big pro IMO since the advantage to using both is unclear to me).

CONS:

1. Anecdotally people seem to prefer the other sorting (this was a complaint when Biome was used originally).
2. Would require a pass of manual lint rule application (but now's the time to do it if we want to do it).

Reply:
I chose to set caseInsensitive: false and included a long comment explicitly stating we can change it later when desired.